patch/Patch.ts

```diff
@@ -0,0 +1,44 @@
+export const DEFAULT_ROW_DELIMITER = "-and-"
+export const DEFAULT_COLUMN_DELIMITER = "-is-"
+
+export class Patch {
+    object: any
+    uriEncodedString: string
+    private rowDelimiter: string
+    private columnDelimiter: string
+    constructor(
+        strOrObjLiteral: string | any,
```

Comment:
I'm wondering if we should narrow down `any` here, maybe to `Record<string, any>` or even something more narrow? (I don't know what the supported types are, just `Record<string, string | number>`?)